### DISCUSSION TITLE
Move folder OneSecondDiary to avoid Google Photos Backup

### DISCUSSION THREAD

**Discussion by recetarioMDWiki** [KyleKun/one_second_diary#128] [Q&A]
Hi all!

I love this app but I wondering if is possible to move the folder with all my "seconds" outside of DCIM. I don't want this videos to take up my backup space on Google Photos.

I don't know if is possible just move the folder or I'll broke the entire app (I gues so...)

Thanks!

**Comment by KyleKun** [comment]
Hi, moving the folder will indeed break the app. I believe you can setup directly in Google Photos app to not upload OneSecondDiary folder.

**Reply by recetarioMDWiki** [reply]
Hey, thanks for your reply 😊
The problem is the folder "One Second Diary" is inside of DCIM, and Google Photos understand that all DCIM contents are the folder "Camera", and I can't split which ones I want to make back ups and which ones no. I can make that with others folders like "WhatsApp Images", "Screenshots", "Instagram", among others. I use Android, by the way.

Thanks so much! 😊
